resources/js/Pages/Dashboard.vue

```diff
@@ -2,18 +2,18 @@
     <app-layout>
         <template #header>
             <h2 class="font-semibold text-xl text-gray-800 leading-tight">
-                <div class="py-12">
-                    <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
-                        <div class="bg-white overflow-hidden shadow-xl sm:rounded-lg">
-                            <a href="users">Users Page</a>
-                            <a href="rooms">Rooms Page</a>
-                            <welcome />
-                        </div>
-                    </div>
-                </div>
+                Dashboard
             </h2>
         </template>
 
+        <div class="py-12">
+            <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
+                <div class="bg-white overflow-hidden shadow-xl sm:rounded-lg">
+                    <a href="users">Users Page</a>
```

Comment:
this href should probably be removed

Reply:
Should be fine since the larvel dashboard should be removed later on